packages/gatsby-theme-store/src/sdk/search/converter/useSearchParamsFromQueryVariables.ts

```diff
@@ -51,6 +52,7 @@ export const useSearchParamsFromQueryVariables = (
     const [base] = pathname.split(selectedFacets[0].value)
 
     return {
+      page: Math.floor(from / pageInfo.size),
```

Comment:
Just so readers can understand that this page is the page size, not the page number like the `SearchProvider`.
```suggestion
      page: Math.floor(from / pageInfo.size), // Page size
```

Reply:
Hold on, this is the number of the page. It's 0 if you are seeing the first page, it's 1 if you are seeing the second page and so on

Reply:
You're right! I'm sorry about that :)